hera_cal/io.py

```diff
@@ -1148,6 +1152,12 @@ def read_hera_hdf5(filenames, bls=None, pols=None, full_read_thresh=0.002,
     inds = {}
     # Read file metadata to size up arrays and sort times
     filenames = _parse_input_files(filenames, name='input_data')
+    
+    if bls is not None:
+        antpairs = [bl[:2] for bl in bls]
+    else:
+        antpairs = None
```

Comment:
Why are we ignoring polarization? If I only want to load one bl for one pol, shouldn't that be allowed?

Reply:
Yeah, you can do that. This is just getting the antpairs for when we check whether all the antpairs are present in the data.